frontend/public/module/k8s/k8s.ts

```diff
@@ -38,8 +39,40 @@ export const groupVersionFor = (apiVersion: string) => ({
   version: apiVersion.split('/').length === 2 ? apiVersion.split('/')[1] : apiVersion,
 });
 
+export const getLatestVersionForCRD = (crdVersions: CRDVersion[], legacyCRDVersion: string) => {
+  const served = crdVersions.filter((version) => version.served);
+  const sorted = served.sort((v1, v2) => {
+    // First sort on major version
+    const majorV1 = _.toNumber(v1.name.match(/\d+/)[0]);
+    const majorV2 = _.toNumber(v2.name.match(/\d+/)[0]);
+    if (majorV1 !== majorV2) {
+      return majorV2 - majorV1;
```

Comment:
This isn't quite right. `v1` should come before `v2beta1` and `v1beta1` should come before `v2alpha1`.

```
     versions is the list of all API versions of the defined custom resource.
     Version names are used to compute the order in which served versions are
     listed in API discovery. If the version string is "kube-like", it will sort
     above non "kube-like" version strings, which are ordered lexicographically.
     "Kube-like" versions start with a "v", then are followed by a number (the
     major version), then optionally the string "alpha" or "beta" and another
     number (the minor version). These are sorted first by GA > beta > alpha
     (where GA is a version with no suffix such as beta or alpha), and then by
     comparing major version, then minor version. An example sorted list of
     versions: v10, v2, v1, v11beta2, v10beta3, v3beta1, v12alpha1, v11alpha2,
     foo1, foo10.
```

Reply:
@spadgett thanks for clarifying this...you're right about the major versions trumping alpha/betas. I didn't code to that, but will fix it.